examples/step-71/step-71.cc

```diff
@@ -1205,8 +1205,9 @@ namespace Step71
         const ConstitutiveParameters &constitutive_parameters)
       : Coupled_Magnetomechanical_Constitutive_Law_Base<dim>(
           constitutive_parameters)
-      , H_components(0)
-      , C_components(Tensor<1, dim>::n_independent_components)
+      , H_components(0) // 'dim' components starting at vector component 0
+      , C_components(Tensor<1, dim>::n_independent_components) // Starting after
+                                                               // H_components
```

Comment:
Would it be more obvious if we wrote
```
  C_components(dim)
```
?

Reply:
I think so - we usually write `FESystem<dim>(FE_Q<dim>(degree), dim)` so writing `dim` matches what people are more likely to write elsewhere.